src/accapi/structs.py

```diff
@@ -1,7 +1,29 @@
+import struct
+
 from .enums import *
 
 _MAX_INT = 2 ** 31 - 1
 
+
+def unpack_next(msg: list, fmt):
```

Comment:
Ideally, this kind of code would reside in the client. #4 addresses this issue with ThreadedSocketReader.